app/client/src/components/pages/Community/components/tabs/Overview.js

```diff
@@ -539,12 +536,15 @@ function SampleLocationsTab({
       stationProviderName: station.properties.ProviderName,
       stationTotalSamples: station.properties.activityCount,
       stationTotalMeasurements: station.properties.resultCount,
+      characteristicGroupResultCount: JSON.stringify(
+        station.properties.characteristicGroupResultCount,
+      ),
```

Comment:
I know this is used on the Monitoring tab (thanks for adding that back in on the Monitoring tab... I see I accidentally removed it in my last PR), but I'm going to remove it here until/unless it's needed on the Overview tab. I'd imagine eventually some code may be shared among the Overview and Monitoring tabs, so we can revisit the best way to do that then.